### PR TITLE
fix(cli,http-client): Properly serialize timeout for multiquery requests through the http client

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: packages/http-client/src/ceramic-http-client.ts [hunk 1]
```diff
@@ -154,6 +154,7 @@ export class CeramicClient implements CeramicApi {
       method: 'post',
       body: {
         queries: queriesJSON,
+        ...{ timeout },
```
Comment (stephhuynh18) [inline packages/http-client/src/ceramic-http-client.ts:157]: Nitpick: I don't think you need to put timeout in an object then spread
```suggestion
        timeout,
```

File: packages/core/src/ceramic.ts [hunk 2]
```diff
@@ -682,7 +682,7 @@ export class Ceramic implements CeramicApi {
    * @param queries - Array of MultiQueries
    * @param timeout - Timeout in milliseconds
    */
-  async multiQuery(queries: Array<MultiQuery>, timeout?: number): Promise<Record<string, Stream>> {
+  async multiQuery(queries: Array<MultiQuery>, timeout = 7000): Promise<Record<string, Stream>> {
```
Comment (stephhuynh18) [inline packages/core/src/ceramic.ts:685]: Nice to have: add default value to the doc comment

**Review by stephhuynh18 — APPROVED** [review]
LGTM just a few optional nitpicks

File: packages/core/src/ceramic.ts (hunk 2 above)
Comment (stbrody) [reply packages/core/src/ceramic.ts:685]: the autogenerated docs already include default values

File: packages/http-client/src/ceramic-http-client.ts (hunk 1 above)
Comment (stbrody) [reply packages/http-client/src/ceramic-http-client.ts:157]: I was trying to avoid sending the field with an explicit value of `undefined` over the wire

**Pull request merged**